lib/igniter/code/common.ex

```diff
@@ -468,95 +468,48 @@ defmodule Igniter.Code.Common do
       v
   end
 
-  def replace_code(zipper, code) when is_binary(code) do
+  def replace_code(%Zipper{} = zipper, code) when is_binary(code) do
```

Comment:
unrelated to this, but we need to remove the places that take a binary and guess that it is code. We have places that accept `{:code, code}`, but we should probably replace it with an "unrealistic value", `{:__code__, code}` or something.